srcpkgs/samba/template

```diff
@@ -125,6 +132,45 @@ smbclient_package() {
 	}
 }
 
+ctdb_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
```

Comment:
Does this really have a hard dependency on `samba`? Is this automatically picked up by shlib dependency resolution?

Reply:
Yes, it needs to depend on `samba` because `ctdbd` spawns `smbd` processes and manages them. `ctdb` can also be configured to manage a `smb.conf`. It seems to me that dependencies are properly tracked, if I try to remove `samba` it doesn't work:
```
$ sudo xbps-remove samba
samba-4.13.2_2 in transaction breaks installed pkg `ctdb-4.13.2_2'
samba-4.13.2_2 in transaction breaks installed pkg `samba-devel-4.13.2_2'
samba-4.13.2_2 in transaction breaks installed pkg `samba-python3-4.13.2_2'
Transaction aborted due to unresolved dependencies.
```